public/components/FeatureAnywhereContextMenu/CreateAnomalyDetector/AddAnomalyDetector.tsx

```diff
@@ -231,6 +286,80 @@ function AddAnomalyDetector({ embeddable, closeFlyout, mode, setMode }) {
     }
   };
 
+  const getEverythingSuccessfulButton = (detectorId, shingleSize) => {
+    return (
+      <EuiText>
+        <p>
+          Attempting to initialize the detector with historical data. This
+          initializing process takes approximately 1 minute if you have data in
+          each of the last {32 + shingleSize} consecutive intervals.
+        </p>
+        {alertingExists(detectorId) ? (
+          <EuiFlexGroup>
+            <EuiFlexItem>
+              <p>Set up alerts to be notifified of any anomalies.</p>
+            </EuiFlexItem>
+            <EuiFlexItem grow={false}>
+              <div>
+                <EuiButton onClick={() => openAlerting(detectorId)}>
+                  Set up alerts
+                </EuiButton>{' '}
+              </div>
+            </EuiFlexItem>
+          </EuiFlexGroup>
+        ) : null}
+      </EuiText>
+    );
+  };
+
+  const alertingExists = () => {
+    try {
+      const uiActionService = getUiActions();
+      uiActionService.getTrigger('ALERTING_TRIGGER_AD_ID');
+      return true;
+    } catch (e) {
+      console.error('No alerting trigger exists', e);
+      return false;
+    }
+  };
+
+  const openAlerting = (detectorId: string) => {
+    const uiActionService = getUiActions();
+    uiActionService
+      .getTrigger('ALERTING_TRIGGER_AD_ID')
+      .exec({ embeddable, detectorId });
+  };
+
+  const handleAssociate = async (detector: DetectorListItem) => {
+    const augmentVisSavedObjectToCreate: ISavedAugmentVis =
+      getAugmentVisSavedObject(detector.id);
+
+    createAugmentVisSavedObject(augmentVisSavedObjectToCreate)
+      .then((savedObject: any) => {
+        savedObject
+          .save({})
+          .then((response: any) => {
+            notifications.toasts.addSuccess({
```

Comment:
here's a way to make the toast wider:
1. Add the following style in the styles.scss file
```
.euiGlobalToastList {
  width: 500px;
}

.successToast {
  width: 450px;
  position: relative;
  right: 25px;
}
```
2. add this line inside the notifictions.toasts.addSuccess...
 `className: "successToast",`

Reply:
![Screenshot 2023-05-25 at 3 31 22 PM](https://github.com/opensearch-project/anomaly-detection-dashboards-plugin/assets/41348518/e7f30d59-b272-404e-a738-b2b4e586092d)

Reply:
Thank you, made the fix